File: yangxj96-starter/yangxj96-starter-security/src/main/java/io/github/yangxj96/starter/security/bean/StoreType.java
package io.github.yangxj96.starter.security.bean;

/**
 * Token 存储方式
 */
public enum StoreType {
    /**
     * 把Token存储到Redis
     **/
    REDIS,
    /**
     * 把Token存储到Db
     **/
    JDBC
}

File: yangxj96-starter/yangxj96-starter-security/src/main/java/io/github/yangxj96/starter/security/bean/StoreType.kt
package io.github.yangxj96.starter.security.bean

/**
 * Token 存储方式
 */
enum class StoreType {
    /**
     * 把Token存储到Redis
     */
    REDIS,

    /**
     * 把Token存储到Db
     */
    JDBC
}
